Front End/Android/app/src/main/java/com/theelite/portal/ui/adapters/SystemsStatusAdapter.kt

```diff
@@ -34,16 +34,14 @@ class SystemsStatusAdapter(private var devices: MutableList<Device>) :
     }
 
     override fun onBindViewHolder(holder: DeviceViewHolder, position: Int) {
```

Comment:
@superDeano was this just as a placeholder or did it actually work? cause it was crashing for the guys